src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -29,15 +29,27 @@ class DebugCommand extends Command
     private $twig;
 
     /**
-     * {@inheritdoc}
+     * @param Environment $twig
      */
-    public function __construct($name = 'debug:twig')
+    public function __construct($twig = null)
     {
-        parent::__construct($name);
+        parent::__construct();
+
+        if (!$twig instanceof Environment) {
+            @trigger_error(sprintf('Passing a command name as the first argument of "%s" is deprecated since version 3.4 and will be removed in 4.0. If the command was registered by convention, make it a service instead.', __METHOD__), E_USER_DEPRECATED);
+
+            $this->setName(null === $twig ? 'debug:twig' : $twig);
```

Comment:
Sure, but who would name this command `0`? 😄 
It won't even work with the symfony base [`Application`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Application.php#L198) and will juste execute the default command.